spec/config.yml

```diff
@@ -11,7 +11,11 @@ DiscoverAndQueuePdrs: &defaults
       IngestGranule: s3://cumulus-test-sandbox-internal/test-npm-integration/workflows/IngestGranule.json
     queues:
       startSF: https://sqs.us-east-1.amazonaws.com/{{AWS_ACCOUNT_ID}}/test-npm-integration-startSF
-ParsePdr: *defaults
+ParsePdr:
+  <<: *defaults
+  meta:
+    <<: *meta_config
+    topic_arn: arn:aws:sns:us-east-1:{{AWS_ACCOUNT_ID}}:test-npm-integration-sftrackerSns-19Z31WEPO3ZEH
```

Comment:
Should we have a hard-coded ARN in here?

Reply:
We should update the tests to use the workflow template from s3.  For now, I have to add it here.

Reply:
@yjpa7145 https://bugs.earthdata.nasa.gov/browse/CUMULUS-513